framework/test/driver/ServiceRegistryTest.cpp

```diff
@@ -88,6 +89,48 @@ void TestUnregisterFix(BundleContext context)
   US_TEST_CONDITION_REQUIRED(!reference.IsConvertibleTo("IBooService"), "Testing for IsConvertibleTo returning false");
 }
 
+void TestServiceReferenceMiscMembers(BundleContext context)
```

Comment:
I would recommend using a separate test point for each member functions rather than clubbing them together.

Reply:
After testing out creating an own test point, I my current approach is slightly more cleaner (because the boilerplate of creating/getting service reference seems to be less - even when I factor that out to a function). Let me know if you still think a change is warranted.